pyo3-ffi/src/compat.rs

```diff
@@ -42,3 +44,16 @@ pub unsafe fn PyDict_GetItemRef(
         -1
     }
 }
+
+#[cfg_attr(docsrs, doc(cfg(all)))]
+#[cfg(Py_3_13)]
+pub use crate::PyList_GetItemRef;
+
+#[cfg(not(Py_3_13))]
```

Comment:
Think the cfg is needed on this item too, because the docs.rs builder will probably be using an older version than 3.13 for a while :)

```suggestion
#[cfg(not(Py_3_13))]
#[cfg_attr(docsrs, doc(cfg(all())))]
```